src/mapping.ts

```diff
@@ -46,6 +47,33 @@ export function handleTransferSingle(event: TransferSingle): void {
   nft.save();
 }
 
+export function handleTransferKnowOrigin(event: KnowOriginTransferEvent): void {
```

Comment:
There is a `handleTransfer` function that does a similar thing. Maybe you can refactor `handleTransferSupeRare` a bit so it will reuse the same code in `handleTransfer`?